Sources/Protocols/Animatable/Animatable.swift

```diff
@@ -499,6 +501,48 @@ fileprivate extension Animatable where Self: UIView {
     }
   }
 
+  func scale(fromX: Double, fromY: Double, toX: Double, toY: Double, configuration: AnimationConfiguration, completion: AnimatableCompletion? = nil) {
+    if fromX.isNaN || fromY.isNaN || toX.isNaN || toY.isNaN {
```

Comment:
I understand your point of view, I prefer guard, but it's up to you 👍 
What about the others? @JakeLin @SD10